### PR TITLE
Outer layer animation update

### PR DESCRIPTION
**Pull request by JackieW001**
## Overview

Update outer layer animation. Now has pulse to indicate when player can lock in.

Deleted some unused assets.

## Changes Made

Planet model has canLockIn() function. Planet node update() function updated to include when layer can be locked in.

## Screenshots

In Design > Example Animation > Outer_layer_Animation_With_Pulse.mov

### CODE REVIEW

File: source/CIPlanetNode.cpp [hunk 1]
```diff
@@ -48,10 +49,12 @@ void PlanetNode::advanceInnerLayerFrame(LayerNode* node){
     node->innerRing->setFrame(innerFrame);
 }
 
-void PlanetNode::advanceOuterLayerFrame(LayerNode* node, bool isLockedIn, bool isLockingIn){
+void PlanetNode::advanceOuterLayerFrame(LayerNode* node, bool isLockedIn, bool isLockingIn, bool canLockIn){
     unsigned int outerFrame = node->outerRing->getFrame();
     if (isLockingIn){
         outerFrame = (outerFrame < OUTER_RING_LOCKIN_START or outerFrame > OUTER_RING_LOCKIN_END) ?  OUTER_RING_LOCKIN_START : outerFrame += 1;
+    } else if (canLockIn){
+        outerFrame = (outerFrame < OUTER_RING_PULSE_START or outerFrame > OUTER_RING_PULSE_END) ?  OUTER_RING_PULSE_START : outerFrame += 1;
```
Comment (henryz2000) [inline source/CIPlanetNode.cpp:57]: maybe we want to switch these two rows sso that they are sequential

**Comment by yds05238** [comment]
Could we make the transition from pulsing back to normal more smoother? In the case where user fails to lock in new layer, the transition seems abrupt especially if the pulsing outer ring is smaller than the normal ring at the time. Is there a way we could reduce the size of the textures and programmatically resize them back? I think the assets are too large for mobile simulator (but works on a real phone).

**Comment by wlong123** [comment]
It looks like the filling in the circle animation now happens after the player completes the lock in. Is this intended b/c I know before it was happening as the player was locking in

**Comment by JackieW001** [comment]
Oof yea, I just tried it again and the lock in should be done while the player is holding onto the planet. The frame numbers are currently incorrect.

**Review by wlong123 — APPROVED** [review]
LGTM

**Pull request merged**